cel/decls.go

```diff
@@ -309,6 +309,32 @@ func MemberOverload(overloadID string, args []*Type, resultType *Type, opts ...O
 	return decls.MemberOverload(overloadID, args, resultType, opts...)
 }
 
+// FunctionDeclOverloadToFunctionOpt converts a list of protobuf CEL FunctionDecl_Overload instances
+// to a list of FunctionOpt instances.
+func FunctionDeclOverloadToFunctionOpt(overloads []*celpb.Decl_FunctionDecl_Overload) ([]FunctionOpt, error) {
```

Comment:
I believe the desired state would be to convert to a `env.Function` directly rather than to a `decls.FunctionDecl` first.